go.mod

```diff
@@ -141,3 +141,5 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/Masterminds/goutils => github.com/Masterminds/goutils v1.1.1
```

Comment:
why add this with replace
why not list?

Reply:
It's used by some third party library so we have no real control on their versions. This was the solution proposed by `snyk` db. Do you see any issue?